modules/stanford_events/templates/components/list/list.html.twig

```diff
@@ -18,7 +18,7 @@
   * - location: Where it is being held.
 #}
 
-{%- set attributes = attributes.addClass(['su-event-list-item', modifier_class]) -%}
+{%- set attributes = attributes.addClass(['su-card', 'su-event-card', 'su-event-list-item', modifier_class]) -%}
```

Comment:
Maybe this is where the event list as cards came from?

Reply:
Ok thanks. I'll review again. I tried to do it that way because another template wasn't sending down the proper classes